Source/WebCore/accessibility/isolatedtree/AXIsolatedTree.cpp

```diff
@@ -181,19 +181,23 @@ void AXIsolatedTree::generateSubtree(AXCoreObject& axObject)
     queueRemovalsAndUnresolvedChanges({ });
 }
 
-AXIsolatedTree::NodeChange AXIsolatedTree::nodeChangeForObject(AXCoreObject& axObject, AttachWrapper attachWrapper)
+std::optional<AXIsolatedTree::NodeChange> AXIsolatedTree::nodeChangeForObject(AXCoreObject& axObject, AttachWrapper attachWrapper)
 {
     ASSERT(isMainThread());
 
-    auto object = AXIsolatedObject::create(axObject, this);
-    NodeChange nodeChange { object, nullptr };
+    // We should never create an isolated object from an ignored object.
+    if (axObject.accessibilityIsIgnored())
```

Comment:
Do we need to worry if an ignored object chnages and it's children are not ignored.. will those children still be processed?

Reply:
No, we don't need to worry about that. The live tree already skips past ignored objects and picks up their children (reference `AccessibilityObject::insertChild`), so they would be picked up by the live-tree walk we do when we build / update the isolated tree.